MARC.js

```diff
@@ -465,6 +483,9 @@ record.prototype.translate = function(item) {
 					if (creatorFields[i] == "100" || creatorFields[i] == "700" ) {
 						creatorObject = ZU.cleanAuthor(authorTab[j]['a'], "author", true)
 					} else {
+						authorTab[j]['a'] = authorTab[j]['a'].replace(/^[\s\u00A0\.\,\/\[\]\:]+/, '')
```

Comment:
There's no cost to this and it's all over the translator anyway so you might as well leave it in, but just a reminder that you only need to escape ]\\/?- in square brackets. See under metacharacters here: http://www.regular-expressions.info/charclass.html